app/src/main/java/com/matmoongi/data/UserProfile.kt

```diff
@@ -0,0 +1,7 @@
+package com.matmoongi.data
+
+data class UserProfile(
+    val id: String,
+    val age: String? = "",
+    val gender: String? = "",
```

Comment:
nullable string 타입의 default 값이 empty string 이라는 건 상당히 어색하네요.

```suggestion
    val age: String = "",
    val gender: String = "",
```
이거나
```suggestion
    val age: String? = null,
    val gender: String? = null,
```
이 일반적일 것 같은데 어떻게 생각하세요?

Reply:
네 nullable을 안하기 위해 ""값을 넣었는데 ?을 까먹고 빼지 않았습니다! 전자로 수정하겠습니다

Reply:
api 명세를 다시 보니 profile이 전체가 아니라 각각 값을 따로 호출해서 받아야 해서 id값만을 받는 dataclass로 수정했습니다.